data-estate-connecter/de_api.php

```diff
@@ -37,11 +46,34 @@ public function estates($params=[], $id=null, $command="data") {
 				$url.='?'.http_build_query($params);
 			}
 		}
+		//echo $url;
 		$request = wp_remote_get($url);
 		$response = wp_remote_retrieve_body($request);
 		return json_decode($response);
 	}
-	public function assets($params=[], $id=null) {
+
+	public function create_asset($params=[], $estate_id) {
```

Comment:
Hi @robbertcha ,

Can you please add some comments about what this function is suppose to do? It says "create" asset... which gives the impression that it's suppose create an asset in the API, but it returns a URL?

Reply:
This is the function I wrote before I went back to Malaysia, but I think we have already found a better solution, because there is not such function in my latest de_api.php, so I just add the double slash to hide the function and push it again.